core/new-gui/src/app/workspace/service/workflow-status/operator-cache-status.service.ts

```diff
@@ -0,0 +1,90 @@
+import { Injectable } from "@angular/core";
+import { WorkflowActionService } from "../workflow-graph/model/workflow-action.service";
+import { WorkflowWebsocketService } from "../workflow-websocket/workflow-websocket.service";
+import { SCHEMA_PROPAGATION_DEBOUNCE_TIME_MS } from "../dynamic-schema/schema-propagation/schema-propagation.service";
+import { debounceTime } from "rxjs/operators";
+import { ExecuteWorkflowService } from "../execute-workflow/execute-workflow.service";
+import { merge } from "rxjs";
+import { JointUIService } from "../joint-ui/joint-ui.service";
+import { environment } from "src/environments/environment";
+
+@Injectable({
+  providedIn: "root"
+})
+export class OperatorCacheStatusService {
+  constructor(
+    private jointUIService: JointUIService,
+    private workflowActionService: WorkflowActionService,
+    private workflowWebsocketService: WorkflowWebsocketService
+  ) {
+    if (!environment.operatorCacheEnabled) {
+      return;
+    }
+    this.registerRequestCacheStatusUpdate();
```

Comment:
the function name is confusing, what do you mean by `RequestCacheStatusUpdate`?  does request has cache?